paramiko/transport.py

```diff
@@ -159,6 +164,8 @@ class Transport(threading.Thread, ClosingContextManager):
         "aes192-cbc",
         "aes256-cbc",
         "3des-cbc",
+        "aes128-gcm@openssh.com",
+        "aes256-gcm@openssh.com",
     )
```

Comment:
**Question**: Why are the new ciphers at the bottom? While I am not a cryptographer, I see these ciphers as recommended to use in SSH because they're extremely strong (sources: [Security Stack Exchange](https://security.stackexchange.com/questions/257670/ssh-server-configuration-best-practices), [Mozilla SSH guidelines](https://infosec.mozilla.org/guidelines/openssh), [ssh-audit hardening guidelines](https://www.ssh-audit.com/hardening_guides.html)).

Reply:
Because there might be some issues with the new cipher suites, consider putting it at the bottom.